src/core/PeakController.cpp

```diff
@@ -37,53 +37,35 @@
 namespace lmms
 {
 
-
 PeakControllerEffectVector PeakController::s_effects;
 int PeakController::m_getCount;
 int PeakController::m_loadCount;
 bool PeakController::m_buggedFile;
 
-
-PeakController::PeakController( Model * _parent,
-		PeakControllerEffect * _peak_effect ) :
-	Controller( ControllerType::Peak, _parent, tr( "Peak Controller" ) ),
-	m_peakEffect( _peak_effect ),
-	m_currentSample( 0.0f )
+PeakController::PeakController(Model * _parent,
+		PeakControllerEffect * _peak_effect) :
+	Controller( ControllerType::Peak, _parent, tr("Peak Controller")),
+	m_peakEffect(_peak_effect),
+	m_currentSample(0.0f)
```

Comment:
... as much as it pains me to preserve bad formatting, the rest of the file is rampant with it, so there's no need to touch more lines than we have to... (there's some deleted lines you could add back too, but GitHub doesn't allow reviews on deleted lines)

```suggestion
PeakController::PeakController( Model * _parent,
		PeakControllerEffect * _peak_effect ) :
	Controller( ControllerType::Peak, _parent, tr( "Peak Controller" ) ),
	m_peakEffect( _peak_effect ),
	m_currentSample( 0.0f )
```

Reply:
I mean, wouldn't it be better to fix more formatting than less?

Reply:
I don't understand this question... What part of PeakController.cpp do you plan on "fixing more" of?